app/transformers/api/v4/user_transformer.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V4::UserTransformer
+  class << self
+    def to_response(user)
+      { 'user' =>
+          Api::V4::Transformer.to_response(user)
+                              .merge('registration_facility_id' => user.registration_facility.id,
+                                     'phone_number' => user.phone_number,
+                                     'password_digest' => user.phone_number_authentication.password_digest)
+                              .except('otp',
+                                      'otp_valid_until',
+                                      'access_token',
+                                      'logged_in_at',
+                                      'role',
+                                      'organization_id') }
+    end
+  end
+end
```

Comment:
I'd love to add specs for this transformer. I've added some reference specs for the V3 transformer in #856.